galsim/compound.py

```diff
@@ -902,3 +905,263 @@ def __setstate__(self, d):
 _galsim.SBFourierSqrt.__setstate__ = lambda self, state: 1
 _galsim.SBFourierSqrt.__repr__ = lambda self: \
         'galsim._galsim.SBFourierSqrt(%r, %r)'%self.__getinitargs__()
+
+
+class RandomWalk(Sum):
+    """
+
+    A class for generating a set of point sources distributed using a random
+    walk.  Uses of this profile include representing an "irregular" galaxy, or
+    adding this profile to an Exponential to represent knots of star formation.
+
+    Random walk profiles have "shape noise" that depends on the number of point
+    sources used.  For example, with 100 points the shape noise is g~0.05, and
+    this will decrease as more points are added.  The profile can be sheared to
+    give additional ellipticity, for example to follow that of an associated
+    disk.
+
+    We use the analytic approximation of an infinite number of steps, which is
+    a good approximation even if the desired number of steps were less than 10.
+
+    The requested half light radius (hlr) should be thought of as a rough
```

Comment:
I used (hlr) with parentheses to show I would be using an abbreviation for this section of docs